charts/super-agent-deployment/templates/uninstall-job.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.cleanupManagedResources -}}
+{{- $uninstallJobName := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "uninstall-job" ) -}}
```

Comment:
in other places I think we have variables directly where they are used, any reason to move them here?

Reply:
We algo have this approach in [some other places](https://github.com/newrelic/helm-charts/blob/master/charts/super-agent/templates/install-job.yaml#L2). Since the variable value is quite long I think is easier to read this way.